src/components/Layout/Layout.stories.tsx

```diff
@@ -110,6 +110,29 @@ export function ScrollableParentFallback() {
   );
 }
 
+export function WithBackgroundColor() {
+  const [selectedTab, setSelectedTab] = useState("lineItems");
+  const contentBgColor = Palette.Gray300;
+  const tabs: TabWithContent[] = [
+    { value: "overview", name: "Overview", render: () => <OverviewExample bgColor={contentBgColor} /> },
+    { value: "lineItems", name: "Line Items", render: () => <ScrollableTableExample bgColor={contentBgColor} /> },
+    { value: "history", name: "History", render: () => <HistoryExample /> },
+  ];
+  return (
+    <TestProjectLayout>
+      <TestHeader title="Change Event - Mud Room" />
+      <div css={Css.pt1.$}>
+        <TabsWithContent
+          selected={selectedTab}
+          tabs={tabs}
+          onChange={(t) => setSelectedTab(t)}
+          contentXss={Css.bgColor(contentBgColor).$}
```

Comment:
Unfortunately to really get this to work as we'd want with tabs, we need to define the same background color on the `<Tab contentXss />` and defined on `<ScrollableContent bgColor />`. Can get into more details if needed, but basically it's due to the Tab's content being moved into the ScrollableContent portal.